src/PostgREST/MediaType.hs

```diff
@@ -1,7 +1,7 @@
 {-# LANGUAGE DeriveAnyClass        #-}
 {-# LANGUAGE DeriveGeneric         #-}
 {-# LANGUAGE DuplicateRecordFields #-}
-
+{-# OPTIONS_GHC -Wno-unused-do-bind #-}
```

Comment:
First time seeing this one, why is it needed?

Reply:
GHC just raises a warning when:

```haskell
P.char '/' -- unused do bind
```

GHC suggest to alternatively use underscore:

```haskell
_ <- P.char '/'
```
This looks kind of ugly. So this option removes that warning. We also use this in:

https://github.com/PostgREST/postgrest/blob/6b4648d2e4ea37e9ebea2b9ab0ee6acaa5a09d4f/src/PostgREST/Config/JSPath.hs#L1